konami/boards.md

```diff
@@ -160,3 +162,21 @@ Konami's own gaming PCs.
     **SSD:** Innodisk 3ME3 2.5" SATA (256GB, DES25-B56D08BC1QC)   
     **SND:** ASUS Xonar AE  
     **Games:** TDJ, UFC
+
+## Bubble System
+
+## System 573
+
+## Twinkle
```

Comment:
This is already documented at the top of the file.

Reply:
Same.